compose/compose_strategy.go

```diff
@@ -49,12 +49,13 @@ func NewOAuth2HMACStrategy(config *Config, secret []byte, rotatedSecrets [][]byt
 	}
 }
 
-func NewOAuth2JWTStrategy(key *rsa.PrivateKey, strategy *oauth2.HMACSHAStrategy) *oauth2.DefaultJWTStrategy {
+func NewOAuth2JWTStrategy(key *rsa.PrivateKey, strategy *oauth2.HMACSHAStrategy, issuer string) *oauth2.DefaultJWTStrategy {
```

Comment:
I could also make `NewOAuth2JWTStrategyWithIssuer`.

Reply:
Yup, that would be preferred!

Reply:
Done.